zookeeper-assembly/pom.xml

```diff
@@ -103,6 +103,10 @@
       <groupId>jline</groupId>
       <artifactId>jline</artifactId>
     </dependency>
+    <dependency>
+      <groupId>ch.qos.logback</groupId>
+      <artifactId>logback-classic</artifactId>
+    </dependency>
```

Comment:
By convention, Apache Accumulo marks all of its assembly dependencies as "optional", just in case somebody actually depends on the tarball out of a Maven repository, it won't resolve all the dependencies that were used to build the tarball. We could have used "provided" instead for that purpose. But ultimately, it doesn't matter if you're not publishing the assembly to Maven. The conventions for dealing with assemblies in Maven repos kind of break all the norms used for jars. Either way, everything bundled in the assembly should be explicitly listed as a dependency, regardless of whatever other scopes/modifiers you put on them.